src/backend/starknet.cairo

```diff
@@ -89,6 +93,33 @@ namespace Starknet {
         let (bytecode_len, bytecode) = IAccount.bytecode(starknet_address);
         return (bytecode_len, bytecode);
     }
+
+    // @notice Populate a Environment with Starknet syscalls
+    func get_env{syscall_ptr: felt*}(
+        origin: model.Address*, gas_price: felt
+    ) -> model.Environment* {
+        alloc_locals;
+        let (block_number) = get_block_number();
+        let (block_timestamp) = get_block_timestamp();
+        let (block_hashes) = alloc();
+        // TODO: fix how blockhashes are retrieved
+        memset(block_hashes, 0, 256 * 2);
+
+        let coinbase = Uint256(
+            0xacdffe0cf08e20ed8ba10ea97a487004, 0x388ca486b82e20cc81965d056b4cdca
+        );
+        return new model.Environment(
+            origin=origin,
+            gas_price=gas_price,
+            chain_id=Constants.CHAIN_ID,
+            prev_randao=Uint256(0, 0),
+            block_number=block_number,
+            block_gas_limit=Constants.BLOCK_GAS_LIMIT,
+            block_timestamp=block_timestamp,
```

Comment:
If we want to stick to EELS -> `gas_limit`, `time` and `number`. Also we are missing `base_fee_per_gas` (also `traces`, `state` and `caller` but I don't think those are important for now).

Reply:
but in KKT1 we did

```
            let mut env = Environment {
                origin: origin.evm,
                gas_price,
                chain_id: get_tx_info().unbox().chain_id.try_into().unwrap(),
                prevrandao: 0,
                block_number: block_info.block_number,
                block_timestamp: block_info.block_timestamp,
                block_gas_limit: constants::BLOCK_GAS_LIMIT,
                coinbase,
                state: Default::default(),
            };
```

and this is what I copied